Lib/fontTools/feaLib/lexer.py

```diff
@@ -45,13 +47,13 @@ def __init__(self, text, filename):
     def __iter__(self):
         return self
 
-    def next(self):  # Python 2
-        return self.__next__()
+    def next(self, comments=False):  # Python 2
```

Comment:
> in normal cases you never call the next method directly

the `__next__()` method is called upon by the `next()` built-in function
https://docs.python.org/3/library/functions.html#next